src/site/content/en/blog/snap-after-layout/index.md

```diff
@@ -0,0 +1,179 @@
+snapped even after changing layout. They will re-evaluate scroll positions after
+changing their layout, and re-snap to the closest snap position if necessary. If
+the scroller was previously snapped to an element that still exists after the
+layout change, then the scroller will try to snap back to it. Pay attention to
+what happens when the layout changes in the following
+[example](https://codepen.io/argyleink/full/YzXyOaX).
+
+<div class="w-columns">
+{% Compare 'worse', 'Snap position lost' %}
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./snap-positions-lost.webm" type="video/webm;">
+    <source src="./snap-positions-lost.mp4" type="video/mp4;">
+  </video>
+</figure>
+
+{% CompareCaption %}
+Rotating a device **does not** preserve the snap positions in Chrome 80.
+{% endCompareCaption %}
+
+{% endCompare %}
+
+{% Compare 'better', 'Snap position preserved' %}
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./snap-positions-preserved.webm" type="video/webm;">
+    <source src="./snap-positions-preserved.mp4" type="video/mp4;">
+  </video>
+</figure>
+
+{% CompareCaption %}
+Rotating a device **does** preserve the snap positions in Chrome 81.
+{% endCompareCaption %}
+
+{% endCompare %}
+</div>
+
+See the [Re-snapping after layout changes
+specification](https://drafts.csswg.org/css-scroll-snap-1/#re-snap) for more
+details.
+
+{% Aside 'caution' %} Snapping is also executed when the page loads. This
+affects the initial scroll offset of scrollers using the scroll snap feature.
+{% endAside %}
+
+## Example: Sticky scrollbars
+With "Snap after layout", developers can implement sticky scrollbars with a few
+lines of CSS:
+```css
+.container {
+  scroll-snap-type: y proximity;
+}
+
+.container::after {
+  scroll-snap-align: end;
+  display: block;
+}
+```
+Want to learn more? See the following [demo chat
+UI](https://codepen.io/argyleink/pen/RwPWqKe) for visuals.
+
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./scroll-snap-bottom.webm" type="video/webm;">
+    <source src="./scroll-snap-bottom.mp4" type="video/mp4;">
+  </video>
+  <figcaption class="w-figcaption">
+    Adding a new message triggers re-snap which makes it stick to the bottom in
+    Chrome 81.
+  </figcaption>
+</figure>
+
+## Future Work
+All re-snapping scroll effects are currently instant; a potential follow-up is
+to support re-snapping with [smooth scrolling
+effects](https://developers.google.com/web/updates/2016/02/smooth-scrolling-in-chrome-49).
+See the [specification issue](https://github.com/w3c/csswg-drafts/issues/4609)
+for details.
+
+## Feedback
+
+Your feedback is invaluable in making re-snapping after layout better, so go on
+and try it out and [let us
+know](https://bugs.chromium.org/p/chromium/issues/detail?id=866127) what you
```

Comment:
```suggestion
and try it out and [let the Chromium engineers
know](https://bugs.chromium.org/p/chromium/issues/detail?id=866127) what you
```

We avoid `us` on web.dev because we want this to be a documentation site for all web developers and we want to eventually get content from other browser vendors. When we use `us` to mean `Chrome engineers` it leaves the impression that this site is only written by Chrome engineers

Reply:
Very good point! Thanks!